package.json

```diff
@@ -3,6 +3,12 @@
   "version": "0.0.001",
   "description": "",
   "main": "server.js",
+  "jshintConfig": {
```

Comment:
All of the different options for javascript linting can be found at http://jshint.com/docs/options/